README.md

```diff
@@ -251,9 +238,15 @@ type Receipt struct {
   # delegation chain from executor to the issuer. This should be 
   # omitted when the executor is the issuer.
   prf     [&UCAN]
+}
 
-  # Signature from the "iss".
-  s       Varsig
+type Receipt struct {
+  trc     Trace
+  sig     Varsig
+}
+ 
+type ReceiptCapsule struct {
+  rct     Receipt (rename "ucan/receipt@0.2.0")
 }
```

Comment:
Arguably this (and Invocation) could be something along the lines of...

```ocaml
type Receipt struct {
  rct Trace (rename "ucan/receipt@0.2.0")
  sig Varsig
}
```

...which removes one level of nesting. That said, it's probably not a terrible idea to also version the signature field?

Reply:
Seems like good intuition?